codegen/codegen_test.go

```diff
@@ -57,13 +56,9 @@ func LocalState(ctx context.Context, t *testing.T, localPath string, opts ...llb
 
 	opts = append([]llb.LocalOption{
 		llb.SharedKeyHint(id),
+		llb.LocalUniqueID(id),
```

Comment:
Would appreciate a sanity check of this change. I found it was needed for codegen tests to pass, because otherwise, without a session ID, BuildKit assigns a random ID to each `local`. I am not really clear on the difference between `SharedKeyHint` and `LocalUniqueID`.

Reply:
Not sure, I think `LocalUniqueID` is different than `LocalID`.    We have a helper func: https://github.com/aaronlehmann/hlb/blob/304e004c45688ea5af8eea22ecd8df99e6a85dad/pkg/llbutil/id.go#L38
so maybe it makes sense to use that like we do [here](https://github.com/aaronlehmann/hlb/blob/304e004c45688ea5af8eea22ecd8df99e6a85dad/pkg/llbutil/id.go#L18-L22)?

Reply:
I chatted with Tonis and it seems like what I'm doing here is correct. Basically this serves as a unique identifier for the data when the session ID isn't available.